src/Sql/SecretsManager/dbo/Tables/ApiKey.sql

```diff
@@ -1,14 +1,15 @@
 ﻿CREATE TABLE [dbo].[ApiKey] (
-    [Id]               UNIQUEIDENTIFIER,
-    [ServiceAccountId] UNIQUEIDENTIFIER NULL,
-    [Name]             VARCHAR(200) NOT NULL,
-    [ClientSecret]     VARCHAR(30) NOT NULL,
-    [Scope]            NVARCHAR (4000) NOT NULL,
-    [EncryptedPayload] NVARCHAR (4000) NOT NULL,
-    [Key]              VARCHAR (MAX) NOT NULL,
-    [ExpireAt]         DATETIME2(7) NULL,
-    [CreationDate]     DATETIME2(7) NOT NULL,
-    [RevisionDate]     DATETIME2(7) NOT NULL,
+    [Id]                    UNIQUEIDENTIFIER,
+    [ServiceAccountId]      UNIQUEIDENTIFIER NULL,
+    [Name]                  VARCHAR(200) NOT NULL,
+    [ClientSecret]          VARCHAR(30) NOT NULL,
+    [HashedClientSecret]    VARCHAR(128) NULL,
```

Comment:
I think we should follow the convention for MasterPassword and use it as a suffix.

```suggestion
    [ClientSecret]          VARCHAR(30) NOT NULL,
    [ClientSecretHash]    VARCHAR(128) NULL,
```

Reply:
Renamed and resolved in https://github.com/bitwarden/server/pull/2943/commits/3c640e6fe71ce8f18864693f0aa63bdf8ab57cdb